client/src/HoverProvider.ts

```diff
@@ -128,42 +149,58 @@ function formatAbout(response: string) {
   // Or
   // |Notation leibnizO A := (discreteO A)
   // |Expands to: Notation iris.algebra.ofe.leibnizO
+  //
+  // Or
+  // |Notation z_to_addr := finz.of_z
+  // |Expands to: Notation cap_machine.addr_reg.z_to_addr
   if (response.match(/not a defined object\./gs) !== null) return;
 
   if (response.startsWith("Notation")) {
     const array = response.split(/\n(?!\s)/gms); // split on newline NOT followed by space
-    let hover = []
-    const match = array[0].match(/Notation\s+(.*?)\s+:=\s+\(/);
+    let hover: vscode.MarkdownString[] = []
+    const match = array[0].match(/Notation\s+(.*?)\s+:=\s*/);
```

Comment:
I like your fix. I have only one question: I wonder if using this regexpr could make the code simpler:
```suggestion
    const match = array[0].match(/Notation\s+(.*?)\s+:=\s*(\()?/);
```
and then testing `match[2]`. Or maybe you could do the entire parsing with a regexp:
```suggestion
    const match = array[0].match(/Notation\s+(.*?)\s+:=\s*((\(?)[^)](\))?)/);
```

But this is just an idea, feel free to ignore it if it doesn't seem helpful.

Reply:
I can use a regexp to figure out if the notation starts with a parentheses, but I can't do the entire parsing with a regexp since you can't parse well-parenthesized expression with regexps.

Reply:
As it stands I think its more legible to have `definition[0] === "("` as the test than to have a more complex regexp